packages/webapp/.env

```diff
@@ -22,7 +22,7 @@ NEXT_PUBLIC_SUBCHAIN_WS_URL = "ws://localhost:3032/v1/subchain/eden-microchain"
 NEXT_PUBLIC_SUBCHAIN_SLOW_MO = "false"
 NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT = "10.00000000 WAX"
 # NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT = "10.0000 EOS"
-NEXT_PUBLIC_ENABLED_WALLETS = "ANCHOR,LEDGER"
+NEXT_PUBLIC_ENABLED_WALLETS = "ANCHOR,LEDGER,SOFTKEY"
```

Comment:
Should we remove Softkey here or leave it as is to help dev and always remember to override in prod?

Reply:
We shouldn't have to override in prod, as there will be a different chainId and so `SOFTKEY` should come back as not available.

Reply:
oh, great!